concordium-cis2/src/lib.rs

```diff
@@ -559,8 +769,9 @@ mod u256_token {
     use super::*;
     use primitive_types::U256;
     #[derive(Debug, Copy, Clone, PartialOrd, Ord, PartialEq, Eq, Default)]
-    #[repr(transparent)]
+    #[cfg_attr(feature = "serde", derive(SerdeSerialize, SerdeDeserialize))]
     #[cfg_attr(docsrs, cfg(feature = "u256_amount"))]
+    #[repr(transparent)]
```

Comment:
Can we add the `serde-serialize/deserialze` to the following parameters in the cis2-library:
```Transfer, TransferParams, OperatorUpdate, UpdateOperator, UpdateOperatorParams```